### PR TITLE
Extract params for checkDB to config.js

### LINKED ISSUE #6: Extract constants for checkDB
Extract constants for checking interval, report interval, etc to a 'constants' file

**Effort: 1 man day**

### PR DESCRIPTION
**Pull request by crbrox**
Fix #6 
Extract params for checkDb to config.js

@fgalan 
@dmoranj 
@jcalderin 

@robertocs :  config.js has changed.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -157,4 +157,18 @@ config.executionsTTL = 1 * 24 * 60 * 60;
...
+/**
+ * CheckDB configuration
+ *
+ * delay Number of milliseconds to check DB connection
+ * @type {Number}
+ *
+ * reportInterval Number of milliseconds to report a problem with DB connection in logs
+ * @type {Number}
+ */
+config.checkDB = {
```
Comment (fgalan) [inline config.js]: Are the new parameters documented in the corresponding .md file?

Comment (crbrox) [inline config.js]: Fixed dcbb159

**Comment by fgalan** [comment]
LGTM (passing the ball to @dmoranj for final LGTM + merge + delete branch)

File: lib/db.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ var async = require('async'),
     logger = require('logops'),
     database,
     orionDb,
-    delay = 2000, reportInterval = 15e3, lastErrorTrace = 0;
+    delay = config.checkDB.delay, reportInterval = config.checkDB.reportInterval, lastErrorTrace = 0;
```
Comment (dmoranj) [inline lib/db.js]: Do this line comply with our Coding style? I think you should add a line brake after each comma.

Comment (crbrox) [inline lib/db.js]: I don't know where our guide is, I usually trust in "reformat code" and linters run by grunt . This passed ... In any case, fixed 1f0b55b

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**